mac/Keyman4MacIM/Keyman4MacIM/KMPackageReader.h

```diff
@@ -0,0 +1,29 @@
+/*
+ * Keyman is copyright (C) SIL International. MIT License.
+ *
+ * KMPackageReader.m
+ * Keyman
+ *
+ * Created by Shawn Schantz on 12/6/21.
```

Comment:
I couldn't figure out how to change the format for this, but it turns out it can be done with the xcode userdefaults. I didn't see anything about date format in the style guide -- if it's not really a standard yet, can we use the MMM version of the month so that it will be even less ambiguous for days like today?

Reply:
I don't know anywhere that does yyyy-dd-mm?

I don't think we've documented date formatting anywhere but I would prefer to go with yyyy-mm-dd, as that is still pretty consistent and matches what we use in some filenames, etc (primary reason yyyy-mm-dd is better than yyyy-MMM-dd is for sorting, which is not relevant for comments, but still...)